sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -17,6 +17,11 @@ import { TokenType } from '@azure/core-amqp';
 import { WebSocketImpl } from 'rhea-promise';
 import { WebSocketOptions } from '@azure/core-amqp';
 
+// @public
+export interface BaseClientOptions {
+    retryOptions?: RetryOptions;
```

Comment:
do we foresee any other options that we want to put in `BaseClientOptions`? If not, then I'd like to avoid introducing a new interface until we have the need for it.

Reply:
I don't know, I can remove the new interface.

@richardpark-msft